DeathNotes.cs

```diff
@@ -504,6 +515,16 @@ private void HandleInconsistencies(ref DeathData data)
                 data.KillerEntityType = CombatEntityType.Helicopter;
                 return;
             }
+
+            // Vehicle Kills
+            if (data.KillerEntityType == CombatEntityType.Player && data.DamageType == DamageType.Generic) {

```

Comment:
Please adjust the brace placement to follow the conventions used here, i.e. move it to the next line.